patroni.py

```diff
@@ -18,6 +18,8 @@
 class Patroni:
 
     def __init__(self, config):
+        assert config["etcd"]["ttl"] > 2 * config["loop_wait"]
```

Comment:
And what about other configuration stores?
Currently we already support ZooKeeper and working on Consul support.